pkgs/tools/networking/ofono/default.nix

```diff
@@ -37,7 +41,7 @@ stdenv.mkDerivation rec {
   ];
 
   configureFlags = [
-    "--with-dbusconfdir=${placeholder ''out''}/etc/dbus-1/system.d"
```

Comment:
Note that this directory is deprecated for this use, and should be `share/dbus-1/system.d`.

> https://dbus.freedesktop.org/doc/dbus-daemon.1.html
> CONFIGURATION FILE section

Reply:
hmm, we have `datadir=/etc` in `lib/pkgconfig/dbus-1.pc`.

Reply:
Weird, perhaps it's because
```
      "--with-systemdsystemunitdir=$(out)/etc/systemd/system"
      "--with-systemduserunitdir=$(out)/etc/systemd/user"
```

in `dbus` in nixpkgs. Doesn't look like those lines changed in forever so we should probably update it.

Reply:
I would have suspect https://github.com/NixOS/nixpkgs/commit/300c9a6c3d73cb486fca6385271acc2cd341ed51